packages/blade/src/components/Alert/_decisions/decisions.md

```diff
@@ -0,0 +1,162 @@
+# Alert
+
+Alerts are messages that communicate information to users about any significant changes or explanations inside the system in a **prominent way**.
+
+This document outlines the API of `Alert` component.
+
+<img src="./alerts-thumbnail.png" width="380" alt="" />
+
+## Design
+
+- [Figma - Alerts](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=6824%3A0)
+
+> **Note**
+>
+> The following needs work:
+>
+> 1. color on banking theme in high contrast
+> 2. a11y contrast ratio for notice alerts in high contrast
+> 3. the borderless variant mentioned in full bleed layout shouldn't have any borders and border radii
+
+## API
+
+Annotated component with props:
+
+![Alert breakdown](./alert-breakdown.png)
+
+Sample usage:
+
+```jsx
+import { Alert, Link } from '@razorpay/blade';
+
+<Alert title="International Payments Only">
+  <Alert.Description>
+    Currently you can only accept payments in international currencies using PayPal. You cannot
+    accept payments in INR (₹) using PayPal. <Link href="https://razorpay.com">Know More</Link>
+  </Alert.Description>
+  <Alert.PrimaryAction
+    onClick={() => {
+      // do something
+    }}
+  >
+    Primary Action
+  </Alert.PrimaryAction>
+  <Alert.SecondaryAction
+    onClick={() => {
+      // do something
+    }}
+    href="https://razorpay.com"
+  >
+    Link
+  </Alert.SecondaryAction>
+</Alert>;
+```
+
+### Alert
+
+We'll expose an `Alert` component with the following API:
+
+| Prop          | Type                                     | Default     | Description                                                                                | Required |
+| ------------- | ---------------------------------------- | ----------- | ------------------------------------------------------------------------------------------ | -------- |
+| `children`    | `string`, Alert sub components           | `undefined` | Body content                                                                               | ✅       |
+| title         | `string`                                 | `undefined` | A brief heading                                                                            |          |
+| isDismissable | `boolean`                                | `true`      | Shows a dismiss button                                                                     |          |
+| contrast      | `high`, `low`                            | `low`       | Can be set to `high` for more prominent look _(not related to a11y)_                       |          |
+| isFullWidth   | `boolean`                                | `false`     | Spans the entire width of container, otherwise max width is restricted to 584px by default |          |
```

Comment:
It would be max-width because the alert shouldn't take more space than the screen size. However on larger screens the alert shouldn't take too much width.

Reply:
I think we should leave the management of alert width on the consumers(developers). Ideally we should create all components assuming that they will occupy all the remaining horizontal space in a container.

Reply:
This can lead to inconsistencies which is one of the primary goals of resolving with blade :)

For example - we'll have to manually ensure across projects that all of them follow a consistent max-width, etc. for all these components